spring-cloud-skipper-server-core/src/main/resources/application.yml

```diff
@@ -41,6 +41,7 @@ server:
 spring:
   main:
     banner-mode: "off"
+    allow-circular-references: true
```

Comment:
Do you mind listing the ticket in the common security project which will allow this to be removed?

Reply:
The ticket has been created so that is close enough to this ask.